### PR TITLE
feat: add entry for @Deferrable to Vaadin usage statistics

### LINKED ISSUE #9540: add an entry for `@Deferred` endpoints in the Vaadin usage statistics
When an app has one or more @Deferred endpoints, it should add a separate entry about that into the usage statistics data like so:
Screenshot 2020-11-28 at 23 22 20

name: endpoint/@Deferred
version: [same as the Flow version]

### PR DESCRIPTION
**Pull request by joheriks**
Fixes #9540.

### CODE REVIEW

File: flow-client/src/main/resources/META-INF/resources/frontend/DeferrableEndpoint.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const $wnd = window as any;
+$wnd.Vaadin = $wnd.Vaadin || {};
+$wnd.Vaadin.registrations = $wnd.Vaadin.registrations || [];
+$wnd.Vaadin.registrations.push({
+    is: '@vaadin/@Deferrable',
+    version: /* updated-by-script */ '6.0-SNAPSHOT'
```
Comment (platosha) [inline flow-client/src/main/resources/META-INF/resources/frontend/DeferrableEndpoint.ts:6]: In addition, `scripts/version.js` should be updated to make it actually updated by script.

**Review by platosha — CHANGES_REQUESTED** [review]

File: flow-client/src/main/resources/META-INF/resources/frontend/DeferrableEndpoint.ts (hunk 1 above)
Comment (joheriks) [reply flow-client/src/main/resources/META-INF/resources/frontend/DeferrableEndpoint.ts:6]: Missed it. Now updates.

Comment (haijian-vaadin) [reply flow-client/src/main/resources/META-INF/resources/frontend/DeferrableEndpoint.ts:6]: Got TS compile erros when trying out the PR.
```
ERROR in [at-loader] ./node_modules/@vaadin/flow-frontend/DeferrableEndpoint.ts:1:7 
    TS2451: Cannot redeclare block-scoped variable '$wnd'.

ERROR in [at-loader] ./target/flow-frontend/DeferrableEndpoint.ts:1:7 
    TS2451: Cannot redeclare block-scoped variable '$wnd'.
```

**Review by haijian-vaadin — CHANGES_REQUESTED** [review]

File: flow-client/src/main/resources/META-INF/resources/frontend/DeferrableEndpoint.ts (hunk 1 above)
Comment (joheriks) [reply flow-client/src/main/resources/META-INF/resources/frontend/DeferrableEndpoint.ts:6]: Good catch, I had somehow dropped the empty export from the commit and it was working randomly when testing it. Now fixed.

**Review by haijian-vaadin — APPROVED** [review]

**Review by platosha — APPROVED** [review]

**Pull request merged**